eng/Versions.props

```diff
@@ -31,11 +31,11 @@
     <MicrosoftCodeAnalysisNetAnalyzersVersion>6.0.0-preview1.21054.10</MicrosoftCodeAnalysisNetAnalyzersVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20623.3</MicrosoftCodeAnalysisTestingVersion>
     <CodeStyleAnalyzerVersion>3.8.0</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>16.9.220</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>16.10.44</VisualStudioEditorPackagesVersion>
     <ILAsmPackageVersion>5.0.0-alpha1.19409.1</ILAsmPackageVersion>
     <ILDAsmPackageVersion>5.0.0-preview.1.20112.8</ILDAsmPackageVersion>
     <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>16.9.150</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
-    <MicrosoftVisualStudioShellPackagesVersion>16.9.30921.310</MicrosoftVisualStudioShellPackagesVersion>
+    <MicrosoftVisualStudioShellPackagesVersion>16.10.0-preview-2-31112-292</MicrosoftVisualStudioShellPackagesVersion>
```

Comment:
It will likely be a few weeks before a 16.10 preview2 build is available on the integration test queue. Should we start a validation insertion for this PR?

Reply:
@JoeRobich before I updated the feedback and merged in main-vs-deps, the integration tests were actually passing!  But I will trigger a validation anyway due to all the upgrades.

Only the LSP ones were failing, but I tested in preview1 (which is being rolled out to scouting tomorrow) and that appeared to fix them.

Reply:
@JoeRobich validation seems fine, jsut hit the same ngen issues main is facing - https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/312719

Reply:
I plan to merge even with the LSP tests failing as they would be fixed on the queue update to 16.10p1 (and only failing in main-vs-deps).